papis/commands/rename.py

```diff
@@ -14,15 +14,15 @@
 - Rename folders for documents whose author is "Rick Astley". You can then either
   enter a new folder name or accept Papis' suggestion. The suggested folder name
   will be generated according to :confval:`add-folder-name` or, if this option
-  isn't set, the current folder name.
+  isn't set, the current folder name. For example:
```

Comment:
```
  isn't set, the current folder name.
```
We're in the examples section, so I think this isn't necessary

Reply:
This was just so the colon makes more sense, but I suppose in this case we can just have a dot here.